app/features/atp/passport/ui/idx/controller.ts

```diff
@@ -116,26 +116,19 @@ const passportValidationMiddleware = [
 ];
 
 const getPassport = (req: Request, res: Response): void => {
-  // if (!req.session.userData.passport) {
-  //   req.session.userData.passport = {};
-  // }
-
-  // const {
-  //   number,
-  //   surname,
-  //   givenNames,
-  //   dob,
-  //   issued,
-  //   expiry,
-  // } = req.session.userData.passport;
-
-  const number = null;
-  const surname = null;
-  const givenNames = null;
+  if (!req.session.identityEvidence) {
+    req.session.identityEvidence = [];
+  }
 
-  const dob = null;
-  const issued = null;
-  const expiry = null;
+  const sessionAttributes =
```

Comment:
I think the plan we had from our group coding session wasn't quite right - this is not looking as simple as I was hoping for.

Let's have a paired session and see if we can improve this section a little bit more.

Reply:
@martin-bucinskas has a new subticket to rework how editing these values should work.